lib/librte_regexdev/Makefile

```diff
@@ -21,6 +21,6 @@ CFLAGS += $(WERROR_FLAGS)
 SRCS-$(CONFIG_RTE_LIBRTE_REGEXDEV) := rte_regexdev.c
 
 # export include files
-SYMLINK-$(CONFIG_RTE_LIBRTE_REGEXDEV)-include += rte_regexdev.h
+SYMLINK-$(CONFIG_RTE_LIBRTE_REGEXDEV)-include += rte_regexdev.h rte_regexdev_core.h
```

Comment:
@orikam 
since we are adding regex device core functions here, commit log should say regex core functions instead of regex driver functions...

Reply:
will fix